.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,9 @@
 <!--- Hello Dagster contributor! It's great to have you with us! -->
 <!-- Make sure to read https://docs.dagster.io/community/contributing -->
 
-## Summary
-<!-- Describe your changes here, include the motivation/context, test coverage, -->
-<!-- the type of change i.e. breaking change, new feature, or bug fix -->
+<!-- Describe your changes here, include the motivation/context  -->
 <!-- and related GitHub issue or screenshots (if applicable). -->
 
+### Test Plan
 
-
-
-## Test Plan
-<!--- Please describe the tests you have added and your testing environment (if applicable). -->
-
-
-
-
-## Checklist
-<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask in our Slack. -->
-
-- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
-- [ ] I have added tests to cover my changes.
\ No newline at end of file
+<!--- Please describe how you verified your changes. -->
```

Comment:
Thoughts on whether we need these comments here (and in the section above)? I usually end up deleting them, because I want the pre-formatted text to look nice and be easy to scan while I'm editing it. I wonder if we can make the section names self-explanatory enough?  E.g. what if we renamed the section headers to "Summary & Motivation" and "How I Tested These Changes"?

Reply:
I would be down with that

Reply:
🙏 yes please, getting rid of the checklist would already be a huge improvement